cypress/support/commands.js

```diff
@@ -225,7 +225,9 @@ Cypress.Commands.add('processDocument', (action, expectedStatus) => {
 
 Cypress.Commands.add('openAdvancedEdit', () => {
   describe('Open the advanced edit overlay via ALT+E shortcut', function() {
-    cy.get('body').type('{alt}E')
+    cy.get('body').type('{alt}N');
```

Comment:
hi @siemiatj , i briefly checked the issue and the file `product_set_documentnote.js`
the product with ID `2005577` is "Convenience Salat 250g"
What makes you think that this test never worked?

Reply:
on my local machine, the test even works right now..

Reply:
Because after firing the keyboard shortcut command never waited for the modal to show up. Look how and when the test fails. I'm pretty sure that if it DID pass before, it's just luck or some weird shenanigans happening because it clearly is really susceptible to timing issues.

I'm not sure why I thought that firing `{alt}N` is safe. Needs fixing.

Reply:
ok, so the  `cy.get('.panel-modal').should('exist');` is the fix

Reply:
i like the command, but i think we need it to be language idependent..
I stumbled over `Cypress.messages` which you create in `index.js`..can we leverage that?
or if needed, we can also make sure that there is an `internalName` property for the whole window.